<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pharmacist_00</class>
 <widget class="QWizardPage" name="Pharmacist_00">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lastLabel">
     <property name="toolTip">
      <string comment="PHARMACIST LAST EDIT TOOLTIP">PHARMACIST LAST</string>
     </property>
     <property name="text">
      <string comment="PHARMACIST LAST EDIT LABEL">PHARMACIST LAST</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lastField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="PHARMACIST LAST EDIT TOOLTIP">PHARMACIST LAST</string>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="firstLabel">
     <property name="toolTip">
      <string comment="PHARMACIST FIRST EDIT TOOLTIP">PHARMACIST FIRST</string>
     </property>
     <property name="text">
      <string comment="PHARMACIST FIRST EDIT LABEL">PHARMACIST FIRST</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="firstField">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="PHARMACIST FIRST EDIT TOOLTIP">PHARMACIST FIRST</string>
       </property>
       <property name="maxLength">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="initialsLabel">
     <property name="toolTip">
      <string comment="INITIALS EDIT TOOLTIP">INITIALS</string>
     </property>
     <property name="text">
      <string comment="INITIALS EDIT LABEL">INITIALS</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLineEdit" name="initialsField">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string comment="INITIALS EDIT TOOLTIP">INITIALS</string>
       </property>
       <property name="maxLength">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lastField</tabstop>
  <tabstop>firstField</tabstop>
  <tabstop>initialsField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
